pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -43,7 +43,9 @@ social:
 
 #top{style: "position: relative; width: 100%; overflow: hidden;"}
 
-  #fullwidth0{style: "z-index: -5; position: absolute; left: 0; top: 0; background-image: url(#{localized_image('/images/homepage_0.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height: 100%; opacity: 1; background-position: 70% 20%"}
+  - first_slide = @country == 'ca' ? '/images/group_girls.jpg' : "#{localized_image('/images/homepage_0.jpg')}"
```

Comment:
since this is already ruby you don't need `"#{`  ... you can just call `localized_image` directly